code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -107,6 +111,24 @@
 			transport containing a nuclear fission explosive, although it is useless without the proper code and authorization disk. While the code was likely found in minutes, the only disk that \
 			can activate this explosive is on your station. Ensure that it is protected at all times, and remain alert for possible intruders."
 
+/datum/game_mode/nuclear/generate_credit_text()
+	var/list/round_credits = list()
+	var/len_before_addition
+
+	if((world.time-SSticker.round_start_time) < (FLUKEOPS_TIME_DELAY)) // If the nukies died super early, they're basically a massive disappointment
+		title_icon = "flukeops"
+
+	round_credits += "<center><h1>The [syndicate_name()] Operatives:</h1></center>"
+	len_before_addition = round_credits.len
+	for(var/datum/mind/operative in syndicates)
+		round_credits += "<center><h2>[operative.name] as a nuclear operative</h2>"
+	if(len_before_addition == round_credits.len)
+		round_credits += list("<center><h2>The operatives blew themselves up!</h2>", "<center><h2>Their remains could not be identified!</h2>")
```

Comment:
I feel like the "blew themselves up" thing should be reserved for if the round ended in a humiliating nuke op defeat (in which the nuke is detonated in the syndicate base that the nuke ops initially spawn in), no?

Reply:
they definitely blow themselves up regardless of if their base gets nuked